src/Sulu/Bundle/ContactBundle/Behat/ContactContext.php

```diff
@@ -22,12 +22,16 @@
  */
 class ContactContext extends BaseContext implements SnippetAcceptingContext
 {
+    private static $emailTypeId = 100;
```

Comment:
Why not simply using the `AutoIncrementSubscriber`? Do we rely on or test against these values?